UPGRADE-1.8.md

```diff
@@ -20,55 +20,33 @@
     ```diff
         parameters:
     +       sylius.security.new_api_route: "/new-api"
-    +       sylius.security.new_api_admin_route: "%sylius.security.new_api_route%/admin"
-    +       sylius.security.new_api_admin_regex: "^%sylius.security.new_api_admin_route%"
-    +       sylius.security.new_api_shop_route: "%sylius.security.new_api_route%/shop"
-    +       sylius.security.new_api_shop_regex: "^%sylius.security.new_api_shop_route%"
+    +       sylius.security.new_api_regex: "^%sylius.security.new_api_route%"
         
         security:
             providers:
     +           sylius_api_admin_user_provider:
     +               id: sylius.admin_user_provider.email_or_name_based
     +           sylius_api_shop_user_provider:
     +               id: sylius.shop_user_provider.email_or_name_based
+    +           sylius_api_admin_and_shop_user_provider:
+    +               chain:
+    +                   providers: [sylius_api_admin_user_provider, sylius_api_shop_user_provider]
```

Comment:
Maybe it would be worth to mention somewhere the edge case with admin user and shop user with the same emails and passwords? Just unit we solve it somehow :)